omdb/views/addTag.ejs

```diff
@@ -0,0 +1,12 @@
+
+<form action="/favorites/<%= id %>/tags" method="POST">
+	<textarea class="form-control" rows="1" name="word" placeholder="Add tag here"></textarea>
+	<button type="submit" id="tagButton">Add</button>
+</form>
+
+<!-- LISTS ALL COMMENTS -->
+ <% tag.forEach(function(tag) {  %>
```

Comment:
Now that we've included our tags with the favorite and passed them to this page, we can iterate over them like this:
```
 <% favorite.tags.forEach(function(tag) {  %>
 <div class="well">
 	<p><%= tag.id %><%= tag.word %></p>
 </div>
 	<% }); %>
```